concourse/docker/pxf-build-base/Dockerfile

```
@@ -15,7 +15,7 @@ RUN apt-get update && \
 RUN cd /tmp && \
     wget -O go.tgz -q https://dl.google.com/go/go1.17.6.linux-amd64.tar.gz && \
     tar -C /usr/local -xzf go.tgz && rm go.tgz && \
-    GOPATH=/opt/go /usr/local/go/bin/go install github.com/onsi/ginkgo/ginkgo
+    GOPATH=/opt/go /usr/local/go/bin/go install github.com/onsi/ginkgo/ginkgo@latest
```

I'm not sure if we want to use the latest version of `ginkgo` since it looks like 2.0 may be coming soon:

> Ginkgo 2.0 is coming soon!
> ==========================
> Ginkgo 2.0 is under active development and will introduce several new features, improvements, and a small handful of breaking changes.
> A release candidate for 2.0 is now available and 2.0 should GA in Fall 2021.  Please give the RC a try and send us feedback!
>   - To learn more, view the migration guide at https://github.com/onsi/ginkgo/blob/ver2/docs/MIGRATING_TO_V2.md
>   - For instructions on using the Release Candidate visit https://github.com/onsi/ginkgo/blob/ver2/docs/MIGRATING_TO_V2.md#using-the-beta
>   - To comment, chime in at https://github.com/onsi/ginkgo/issues/711

Looking at the [`go.mod`](https://github.com/greenplum-db/pxf/blob/branch-5.16/cli/go/src/pxf-cli/go.mod) for PXF 5.16.x, we might want to stick with `v1.12.0`:

```suggestion
    GOPATH=/opt/go /usr/local/go/bin/go install github.com/onsi/ginkgo/ginkgo@v1.12.0
```